### DISCUSSION TITLE
Partial Lightning network payments for Payment Requests

### DISCUSSION THREAD

**Discussion by jacobburrell** [btcpayserver/btcpayserver#5433] [Ideas / Feature Requests]
If a user wants to make a partial payment for a payment request, it is often better to do so with the LN, but LN invoices in payment requests seem to always have the full amount in the invoice, which doesn't allow the amount to be modified.

That is great if the user is able to pay the full amount, but if partial payments are possible on the chain where they create extra UTXOs, they probably should be possible with the LN which avoids them.

Better to have 5-15 partial payments for a payment request rather than 5-15 partial on-chain txs paying the payment request.

**Comment by dennisreimann** [comment]
You can use the "Allow payee to create invoices in their own denomination" option for this. Actually, denomination isn't correct, because this is for letting the customer specify the amount (fixing that wording issue in #5413) — does this option work for your use case?
